apps/api/src/app/shared/helpers/content.service.ts

```diff
@@ -77,6 +87,10 @@ export class ContentService {
     }
   }
 
+  private isSystemVariable(variableName: string): boolean {
+    return TemplateSystemVariables.includes(variableName.includes('.') ? variableName.split('.')[0] : variableName);
```

Comment:
```suggestion
const variable = variableName.includes('.') ? variableName.split('.')[0] : variableName
    return TemplateSystemVariables.includes(variable);
```

I feel this will be much clearer to read

Reply:
I am actually not sure I agree🙈 For me its easier having that context in the same line 
But I don't mind changing it if you feel strongly about it :) @BiswaViraj

Reply:
No, it's okay, just a personal preference